fix: Updated index.mjs file to be modified for Apollo subgraph Federation 2 features (#126)
aws/amazon-neptune-for-graphql: CHANGELOG.md

  option ([#117](https://github.com/aws/amazon-neptune-for-graphql/pull/117))
* Fixed invalid schema generation when AWS AppSync scalar types are used in 
  an input 
  schema ([#118](https://github.com/aws/amazon-neptune-for-graphql/pull/118))

> * Updated index.mjs file to be modified for Apollo subgraph Federation 2 

Reviewer: ```suggestion
* Updated Apollo subgraph to allow use of Federation 2 features.
```

  an input 
  schema ([#118](https://github.com/aws/amazon-neptune-for-graphql/pull/118))
* Fixed duplicated nodes and edges from nodes with 

>   multi-labels ([#125](https://github.com/aws/amazon-neptune-for-graphql/pull/125))
> * Updated Apollo subgraph to allow use of Federation 2 features.
>   features ([#126](https://github.com/aws/amazon-neptune-for-graphql/pull/126))

Reviewer: Sorry, messed up the conflict resolution merge.

```suggestion
  multi-labels ([#125](https://github.com/aws/amazon-neptune-for-graphql/pull/125))
* Updated Apollo subgraph to allow use of Federation 2 features.
  ([#126](https://github.com/aws/amazon-neptune-for-graphql/pull/126))
```
